### ISSUE TITLE
[Web] RUM stopped working on 16-05-2024

### ISSUE THREAD

**Issue by Arthurius** [DataDog/dd-sdk-flutter#616]
### Describe the bug

For all our products running on Flutter web (4 different projects), all sessions stopped coming in. 

![Screenshot 2024-05-24 at 10 17 01](https://github.com/DataDog/dd-sdk-flutter/assets/4608671/1defb4f5-8a9c-44c7-8e20-8b8f325af1eb)

I do see a new error popping up in the browser console:

![Screenshot 2024-05-24 at 10 10 41](https://github.com/DataDog/dd-sdk-flutter/assets/4608671/eff9072f-95b5-4fe8-a4e6-9fde4dd198b2)

### Reproduction steps

Just look at the session explorer for any Flutter app running on web.

### Affected SDK versions

2.4.0 and 2.5.0

### Did you confirm if the latest SDK version fixes the bug?

Yes

### Flutter Version

3.19.6

### Other relevant information

Everything has been working for months. It stopped working for all our products on the same data, without any deployments or changes from our side.

**Comment by Arthurius** [comment]
These are the screenshots I meant to add
![Screenshot 2024-05-24 at 10 17 01](https://github.com/DataDog/dd-sdk-flutter/assets/4608671/1defb4f5-8a9c-44c7-8e20-8b8f325af1eb)

![Screenshot 2024-05-24 at 10 10 41](https://github.com/DataDog/dd-sdk-flutter/assets/4608671/eff9072f-95b5-4fe8-a4e6-9fde4dd198b2)

**Comment by Arthurius** [comment]
Interesting find: when I change
```
  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/us1/v5/datadog-logs.js"></script>
  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/us1/v5/datadog-rum-slim.js"></script>
```
from v5 to v4, sessions start coming in again..
So it seems to be a bug in the v5 scripts

**Comment by fuzzybinary** [comment]
Hi @Arthurius,

Thanks for pointing this out. I'll see if I can get a fix up for it soon. Switching to v4 may work for you if you're only using logging, but might cause other errors if you use RUM.

**Comment by Arthurius** [comment]
Hi @fuzzybinary, 
Thanks for checking!
Switching is indeed not a valid option. I just switched our dev environment to v4 to test it out.

I hope the issue can get resolved soon.

**Comment by fuzzybinary** [comment]
Hey @Arthurius ,

What are you setting your `DatadogConfiguration.version` to?  My simple checks aren't reproducing the issue.

**Comment by fuzzybinary** [comment]
Nevermind, I reproduced, and my guess is you're not setting `version` in the configuration. If you're looking for a quick fix while we work on an official fix, set `version` and `service` in your configuration to non-null values and you'll be back up and running again.

**Comment by Arthurius** [comment]
Thanks, I've set both the version and the service. Now it works.

**Comment by fuzzybinary** [comment]
The Browser SDK team is now allowing `null` as possible values for service and version. A new version should be rolled out to CDNs in the near future.

**Comment by BenoitZugmeyer** [comment]
👋 Just to let you know, this has been fixed in the latest Browser SDK release [v5.19.0](https://github.com/DataDog/browser-sdk/blob/main/CHANGELOG.md#v5190)
